lib/async_hooks.js

```diff
@@ -412,11 +397,17 @@ function emitDestroyS(asyncId) {
 
 
 function emitDestroyN(asyncId) {
+  if (async_hook_fields[kDestroy] === 0) return;
```

Comment:
Re: https://github.com/nodejs/node/pull/13419/files#r119971593 why was this added?